tests/test_vector_sqlite.py

```
@@ -0,0 +1,730 @@
+                # Store reference to the database connection
+                db_connection = adapter._db
+
+                # Explicitly close the adapter
+                adapter.close()
+
+                # Verify the database connection is closed by trying to use it
+                # This should raise an error if the connection is closed
+                try:
+                    db_connection.execute("SELECT 1")
+                    # If we get here, the connection is still open (which is unexpected)
+                    assert False, "Database connection should be closed"
```

Ruff (0.12.2)

Do not `assert False` (`python -O` removes these calls), raise `AssertionError()`

Replace `assert False`

(B011)